Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -409,18 +408,18 @@ bool MediaPlayerPrivateGStreamerBase::handleSyncMessage(GstMessage* message)
                 return false;
             }
             GST_DEBUG("handling drm-preferred-decryption-system-id need context message");
+            LockHolder lock(m_protectionMutex);
             std::pair<Vector<GRefPtr<GstEvent>>, Vector<String>> streamEncryptionInformation = extractEventsAndSystemsFromMessage(message);
             GST_TRACE("found %" G_GSIZE_FORMAT " protection events", streamEncryptionInformation.first.size());
             Vector<uint8_t> concatenatedInitDataChunks;
             unsigned concatenatedInitDataChunksNumber = 0;
-            String eventKeySystemIdString;
+            const char* eventKeySystemId = nullptr;
```

Comment:
If want to maintain this like this, I'm ok, but in staging-emev3 this will remain a String. Feel free to leave it like this or change it. I'm ok either way.